frontend-ssr/pages/search.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Small nitpick: let's use PascalCase for the component name, ie `Search.tsx` and `Search.module.scss`.
Then also add another `index.ts` file in the same folder which exports that component so we can directly import it like this `import Search from 'components/Search` instead of `import Search from 'components/search/Search'`

Reply:
For this pattern have a look at `Header`, `Footer`, etc. It's newly introduced, I saw it used by @catalinpopusoi  and I think it's a good idea which makes for cleaner imports.